logic/src/commonMain/kotlin/com/wire/kalium/logic/data/event/EventMapper.kt

```diff
@@ -54,6 +59,13 @@ class EventMapper(private val idMapper: IdMapper) {
         eventContentDTO.message
     )
 
+    private fun newConnection(
```

Comment:
One question, this event will handle only newConnections ? Or this mapper will be taking care of everything related to connections ? For example, accepting, ignoring, etc. 🤔 

If the case is the latter (which I think it is) We could rename to `connectionUpdated` maybe =)

Reply:
It should handle all connections, good idea for renaming 👍